src/views/Dissolution/DissolutionReviewConfirm.vue

```diff
@@ -4,7 +4,7 @@
     <section class="mt-10">
       <header>
         <h2>Review and Confirm</h2>
-        <p id="intro-text" class="mt-2">
+        <p class="mt-4 mb-6">
```

Comment:
We use these same classes in all headers on this page, so the whitespace now looks good :)